uptane-standard.md

```diff
@@ -717,7 +718,7 @@ An ECU version report is a metadata structure that MUST contain the following in
   * The filename, length, and hashes of its currently installed image (i.e. the non-custom targets metadata for this particular image)
   * An indicator of any detected security attack
   * The latest time the ECU can verify at the time this version report was generated
-  * A cryptographic nonce
+  * Either a cryptographic nonce or a counter. This value MUST change each update cycle.
```

Comment:
Why does it need to be cryptographic?

Also, could you please explain what this nonce or counter is actually used for?

Finally, is there any reason to use a nonce instead of a counter, other than for backwards-compatibility?